app/components/live-sidebar.hbs

```diff
@@ -1,6 +1,22 @@
-<div data-test-sidebar class='sidebar {{if this.isSideBarOpen "open" "close"}}'>
-  <div data-test-sidebar-participants class='sidebar__participants'>
-    <h4>Participants</h4>
+<div
+  data-test-sidebar
+  class='sidebar
+    {{if this.isSideBarOpen "open" "close"}}
+    {{if @isExpanded "sidebarCloses" "sidebarOpens"}}'
+>
+  <button
+    class='openbtn {{if @isExpanded "closeBtn"}}'
+    type='button'
+    {{on 'click' this.sidebarPanel}}
+  >&#9776;</button>
```

Comment:
We're already using Iconify icons, please use this for consistency

Reply:
done